File: fix_profile.js
import fs from 'fs';

let code = fs.readFileSync('src/screens/ProfileScreen.tsx', 'utf8');

// Typography
code = code.replace(/typography\.fontSize\.xs/g, 'typography.fontSizeXS');
code = code.replace(/typography\.fontSize\.sm/g, 'typography.fontSizeSM');
code = code.replace(/typography\.fontSize\.base/g, 'typography.fontSizeMD');
code = code.replace(/typography\.fontSize\.md/g, 'typography.fontSizeMD');
code = code.replace(/typography\.fontSize\.lg/g, 'typography.fontSizeLG');
code = code.replace(/typography\.fontSize\.xl/g, 'typography.fontSizeXL');

code = code.replace(/typography\.fontWeight\.light/g, 'typography.fontWeightLight');
code = code.replace(/typography\.fontWeight\.regular/g, 'typography.fontWeightRegular');
code = code.replace(/typography\.fontWeight\.medium/g, 'typography.fontWeightMedium');
code = code.replace(/typography\.fontWeight\.semibold/g, 'typography.fontWeightSemiBold');
code = code.replace(/typography\.fontWeight\.bold/g, 'typography.fontWeightBold');

// Colors
code = code.replace(/colors\.text\.primary/g, 'colors.textPrimary');
code = code.replace(/colors\.text\.secondary/g, 'colors.textSecondary');
code = code.replace(/colors\.text\.tertiary/g, 'colors.textMuted');
code = code.replace(/colors\.cardBorder/g, 'colors.border');
code = code.replace(/colors\.card/g, 'colors.cardBackground');
code = code.replace(/colors\.error/g, 'colors.danger');
code = code.replace(/colors\.white/g, 'colors.background');

fs.writeFileSync('src/screens/ProfileScreen.tsx', code);

File: fix_profile.py
import re

path = '/data/.openclaw/workspace/projects/Academic_Aide/academic-aide/src/screens/ProfileScreen.tsx'
with open(path, 'r') as f:
    code = f.read()

# Typography
code = code.replace('typography.fontSize.xs', 'typography.fontSizeXS')
code = code.replace('typography.fontSize.sm', 'typography.fontSizeSM')
code = code.replace('typography.fontSize.base', 'typography.fontSizeMD')
code = code.replace('typography.fontSize.md', 'typography.fontSizeMD')
code = code.replace('typography.fontSize.lg', 'typography.fontSizeLG')
code = code.replace('typography.fontSize.xl', 'typography.fontSizeXL')

code = code.replace('typography.fontWeight.light', 'typography.fontWeightLight')
code = code.replace('typography.fontWeight.regular', 'typography.fontWeightRegular')
code = code.replace('typography.fontWeight.medium', 'typography.fontWeightMedium')
code = code.replace('typography.fontWeight.semibold', 'typography.fontWeightSemiBold')
code = code.replace('typography.fontWeight.bold', 'typography.fontWeightBold')

# Colors
code = code.replace('colors.text.primary', 'colors.textPrimary')
code = code.replace('colors.text.secondary', 'colors.textSecondary')
code = code.replace('colors.text.tertiary', 'colors.textMuted')
code = code.replace('colors.cardBorder', 'colors.border')
code = code.replace('colors.card', 'colors.cardBackground')
code = code.replace('colors.error', 'colors.danger')
code = code.replace('colors.white', 'colors.background')

with open(path, 'w') as f:
    f.write(code)
